docs/docs/docs/01-core/openapi/01-technologies.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Technologies
+slug: /openapi/technologies
+tags:
+  - openapi
+  - technologies
+  - dependencies
+---
+
+# Open API
+
+This section provides an overview of the technologies used in the OpenAPI package
+
+---
+
+## Tools
+
+The OpenAPI package uses the same tools as the rest of the Strapi monorepo packages:
+
+| Tool     | Purpose         |
+| -------- | --------------- |
+| Jest     | Unit testing    |
+| Rollup   | Build           |
+| Prettier | Code formatting |
+| ESLint   | Code linting    |
+
+## Dependencies
+
+The package focuses on using a limited set of external dependencies to keep a light footprint.
+
+---
+
+### [Zod to OpenAPI](https://github.com/asteasolutions/zod-to-openapi)
+
+> A library that uses [zod schemas](https://github.com/colinhacks/zod) to generate Open API Swagger documentation.
+
+This library is used to transform `zod` schema gotten from Strapi routes objects into valid OpenAPI Schema.
+
```

Comment:
```suggestion
This library is used to transform `zod` schemas based on Strapi routes into valid a OpenAPI Schema.

```

Reply:
I was mainly thinking that "gotten from" isn't grammatically correct (sorry to be that guy 🫣)

"used to transform a `zod` schema generated from Strapi routes into a valid OpenAPI Schema."

Does that work?

Reply:
Haha, I don't mind being corrected on my English

> generated from Strapi routes

The thing is, we're not generating anything, schemas are manually defined, and we're just getting that from the route, hence my confusion. (I'm giving you the final word on that one, just wanted to make sure we understood the same thing)